src/components/forecast/ForecastDetail.js

```diff
@@ -0,0 +1,110 @@
+import { useParams } from 'react-router-dom';
```

Comment:
- [ ] Well done on your efforts thus far. Please note, however, that as a requirement there must be a 'Back' button present on the details page. I can see that you have implemented the functionality to the logo of the web app; however, this will not suffice and a dedicated 'Back' button is necessary to meet the requirements.

Reply:
Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/210902115-536acd02-7632-4f37-b392-a1ee77b503e5.png"> | <img src="https://user-images.githubusercontent.com/69011963/210902117-0149d822-d53a-4497-997a-37d0b9f26f87.png" width="1440"/>